helpers/requestMethods.js

```diff
@@ -0,0 +1,30 @@
+const { EndPoints } = require('../Constants/EndPoint');
+
+const endPoints = EndPoints();
+
+async function getList(requestContext, endpoint) {
```

Comment:
why you have named this method as **getList** ?

Reply:
Addressed